src/Controls/src/Core/Platform/Windows/CalendarDatePickerExtensions.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.UI.Xaml.Controls;
+
+namespace Microsoft.Maui.Controls.Platform
+{
+	public static class CalendarDatePickerExtensions
+	{
+		public static void UpdateText(this CalendarDatePicker platformDatePicker, IDatePicker datePicker, TextTransform tranform)
+		{
+			// TODO: Implement TextTransform property on Windows.
```

Comment:
Has this property been implemented in Xamarin.Forms?, What should we do on Windows?

We can access the TextBlock showing the Date , and apply the transformation to the Control text. Opinions?

`<TextBlock x:Name="DateText" Grid.Column="1" Foreground="{ThemeResource CalendarDatePickerTextForeground}" HorizontalAlignment="Left" Padding="12, 0, 0, 2" Grid.Row="1" Text="{TemplateBinding PlaceholderText}" VerticalAlignment="Center"/>`